src/Utility/LogStream.C

```diff
@@ -62,8 +62,10 @@ void utl::LogStream::addExtraLog(std::shared_ptr<std::ostream> extra, bool clear
 int utl::LogStream::precision(int streamsize)
 {
   int result = streamsize;
-  if (m_out)
-    result = m_out->precision(streamsize);
+  if (m_out) {
+    result = m_out->precision();
```

Comment:
Don't remember what troubles you mentioned and when, but this will change the behavior of this method. It will now only set the precision for the extra streams `m_extra` and not the main one `m_out`. Gues you need like this:
```
  int result = streamsize;
  if (m_out) {
    result = m_out->precision();
    m_out->precision(streamsize);
  }
  for (auto it : m_extra)
    it->precision(streamsize);
```